### PR TITLE
fix pool select modal search

### PR DESCRIPTION
**Pull request by pingustar**
#360

### CODE REVIEW

File: src/components/selectPoolModal/SelectPoolModalContent.tsx [hunk 1]
```diff
@@ -9,6 +9,9 @@ interface Props {
 }
 export const SelectPoolModalContent = ({ pools, onSelect }: Props) => {
   const [search, setSearch] = useState('');
+  const filteredPools = pools.filter((pool) =>
+    pool.reserves[0].symbol.toLowerCase().includes(search.toLowerCase())
```
Comment (zavelevsky) [inline src/components/selectPoolModal/SelectPoolModalContent.tsx:13]: is `pool.reserves.length` always > 0?

**Review by zavelevsky — APPROVED** [review]
LGTM, just see if you need to check pool.reserves.length before doing pool.reserves[0].symbol

File: src/components/selectPoolModal/SelectPoolModalContent.tsx (hunk 1 above)
Comment (pingustar) [reply src/components/selectPoolModal/SelectPoolModalContent.tsx:13]: yeah, each pool must have 2 reseves.

pool.reserves[0] => tkn reserve
pool.reserves[1] => bnt reserve

**Comment by pingustar** [comment]
> LGTM, just see if you need to check pool.reserves.length before doing pool.reserves[0].symbol

yeah, each pool must have 2 reseves.

pool.reserves[0] => tkn reserve
pool.reserves[1] => bnt reserve

**Pull request merged**